.goreleaser.yml

```diff
@@ -15,10 +15,10 @@ builds:
   ldflags: -s -w -X main.version={{.Version}}
   binary: thermal-uploader
 
-nfpm:
-  vendor: The Cacophony Project
+nfpms:
+- vendor: The Cacophony Project
   homepage: http://cacophony.org.nz/
-  maintainer: Menno Finlay-Smits <menno@cacophony.org.nz>
+  maintainer: Cacophony Developers <coredev@cacophony.org.nz>
```

Comment:
Thanks. This should become the standard value for this field in all our packages.